pkgs/development/libraries/qt-5/modules/qtmultimedia.nix

```
@@ -15,8 +15,8 @@ qtModule {
   pname = "qtmultimedia";
   qtInputs = [ qtbase qtdeclarative ];
   nativeBuildInputs = [ pkg-config ];
-  buildInputs = [ gstreamer gst-plugins-base libpulseaudio ]
-    ++ lib.optional (stdenv.isLinux) alsa-lib
+  buildInputs = [ gstreamer gst-plugins-base ]
+    ++ lib.optionals stdenv.isLinux [ alsa-lib libpulseaudio ]
```

Preserving the order of arguments is necessary to avoid a mass rebuild.

```suggestion
    ++ lib.optionals stdenv.isLinux [ libpulseaudio alsa-lib ]
```